core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -551,6 +551,16 @@ public static void setUpdateTimeOfDay(int hourOfDay, int minute) {
         restartUpdateAlarm(false);
     }
 
+    public static boolean shouldShowOnboarding(String location) {
+        String key = "onboarding_" + location;
+        if (prefs.getBoolean(key, true)) {
+            prefs.edit().putBoolean(key, false).apply();
```

Comment:
this method knows too much - it is kind of a strange side effect
I'd prefer if we had separate method for acknowledging that the onboarding was done and should not be shown again